src/components/common/Card.jsx

```diff
@@ -100,18 +101,20 @@ const mockdata = {
   createdAt: '2023-10-31T09:58:47.272896Z',
 };
 
-function CardAdd(isEditCard) {
+function CardAdd({ data, isEditCard }) {
   return (
     <Styled.CardContainer isEditCard={isEditCard}>
-      <IconButton shape="plus" />
+      <Link to={`/post/${data.id}/message`}>
+        <IconButton shape="plus" />
+      </Link>
```

Comment:
@joonyoungchoi0801 님! 이렇게 버튼클릭시 다른 곳으로 이동하고 싶을 때, 
Link 태그로 감싸는 방법도 있지만 기존 버튼은 그대로 두고 감쌀필요없이
```suggestion 
     const navigate = useNavigate()
     <IconButton shape="plus" onClick={() => navigate(`/post/${data.id}/message`)} />
```
이런식으로 `useNavigate()`를 사용해주시면 좋습니다!